app/code/Magento/InventoryIndexer/Model/GetInvalidationRequired.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryIndexer\Model;
```

Comment:
Seems this class is better to move to Resource namespace.

Reply:
@RomaKis done.